tests/backend/test_design_step_bbox.py

```
@@ -0,0 +1,99 @@
+    return step_path.read_bytes(), expected
+
+
+@pytest.fixture
+def cad_service_stub(
+    monkeypatch: pytest.MonkeyPatch, sample_step_payload: Tuple[bytes, Dict[str, float]]
+) -> Dict[str, float]:
+    """Mock the CAD service call to return a known STEP payload."""
+
+    step_bytes, expected_dims = sample_step_payload
+
+    async def _mock_request(self, method: str, url: str, *args, **kwargs) -> httpx.Response:  # noqa: ANN001
```

Ruff (0.14.1)

Unused function argument: `self`

(ARG001)